src/templates/docs/forms/index.html

```diff
@@ -41,9 +41,15 @@ <h4 class="f4 mv0 fw6 dib mr4"><%= name %></h4>
         <h2 class="f3 bold">Examples</h2>
         <form>
           <label for="inputReset">
-            <code class="f6 db">input-reset</code>
+            <code class="f6 db gray">.input-reset</code>
           </label>
-          <input id="inputReset" type="text" class="input-reset">
+          <input id="inputReset" type="text" class="input-reset mt2">
+        </form>
+        <form class="mt4">
```

Comment:
Added in an example of `input-invisible` as it felt right to have all available classes as examples, and added placeholder to field to show there is something there